ui/lib/core/addon/components/replication-page.js

```diff
@@ -1,6 +1,35 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import layout from '../templates/components/replication-page';
 
+const MODE = {
+  dr: 'Disaster Recovery',
+  performance: 'Performance',
+};
+
 export default Component.extend({
   layout,
+  mode: computed('model', function() {
+    let mode = this.model.rm.mode;
+    return MODE[mode];
+  }),
+  dr: computed('model', function() {
+    let dr = this.model.dr;
+    if (!dr) {
+      return false;
+    }
+    return dr;
+  }),
+  isDisabled: computed('dr', function() {
+    if (this.dr.mode === 'disabled' || this.dr.mode === 'primary') {
```

Comment:
this is a nit, but since the conditional here only applies to `ReplicationPage` components that are rendering a DR secondary dashboard (i.e. not a `ReplicationPage` rendering a Performance Secondary dashboard), i'd add a comment saying so. that way when you use this component next you'll know that you need to change the conditional to account for both situations. up to you!